client/src/util/pages.jsx

```diff
@@ -48,5 +49,10 @@ export const pages = {
       component: <div />,
       path: '/permission request',
```

Comment:
I am pretty sure this is an invalid path, can we change it to '/permission_request'?